.storybook/theme.js

```diff
@@ -0,0 +1,22 @@
+import { create } from '@storybook/theming/create';
+
+export default create({
+  base: 'dark',
+  brandTitle: 'CFPB Design System React',
+  brandImage: '../src/assets/images/cfpb-seal.png',
+  brandUrl: 'https://cfpb.github.io/design-system/',
+  brandTarget: '_blank',
+
+  // Typography
+  fontBase: '"Avenir Next", Arial ,sans-serif',
+  textInverseColor: '#ffffff',
+
+  // Sidebar
+  appBg: '#101820', // Background
+  app: '#101820', // Background
+  colorPrimary: '#ff9e1b', // Folder icon
+  colorSecondary: '#0072ce', // onHover, Story icon, Active panel (ex. Addons)
+
+  // Toolbars
+  barTextColor: 'white'
```

Comment:
How would you feel about installing the [dark mode addon](https://github.com/hipstersmoothie/storybook-dark-mode) to let users control if they want a dark or light background? The [theme configuration](https://github.com/hipstersmoothie/storybook-dark-mode#configuration) looks pretty similar.

Reply:
Seems sensible, good catch.

Reply:
Updated to implement light/dark theme switching. Screenshots in [PR description](https://github.com/cfpb/design-system-react/pull/159#issue-1862029102).